core/src/main/res/layout/activity_history.xml

```diff
@@ -5,7 +5,23 @@
   android:layout_height="match_parent"
   android:fitsSystemWindows="true">
 
-  <include layout="@layout/layout_standard_app_bar" />
+  <com.google.android.material.appbar.AppBarLayout xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/app_bar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:showIn="@layout/activity_help">
+
+    <include layout="@layout/layout_toolbar" />
+
+    <Switch
```

Comment:
this might look better aligned right?
There needs to be padding around this view
The AppBar backgroundColor should be the same as the Toolbar ie black

Reply:
@macgills done.

Reply:
I meant move it to the right of the screen, the switch is next to invisible on the dark background and it needs more padding around the view, the bottom of the appbar is too close to the switch.
You should try styling the switch with a theme, see if `app:theme="@style/ThemeOverlay.MaterialComponents.Dark` works

Reply:
On it.

Reply:
Also the trash icon is not visible

Reply:
There are also lint errors
```
  /home/runner/work/kiwix-android/kiwix-android/core/src/main/res/layout/activity_history.xml:23: Error: To support older versions than API 17 (project specifies 15) you should also add android:paddingLeft="@dimen/switch_text_padding" [RtlCompat]
212
        android:paddingStart="@dimen/switch_text_padding"
213
        ~~~~~~~~~~~~~~~~~~~~
214
  /home/runner/work/kiwix-android/kiwix-android/core/src/main/res/layout/activity_history.xml:23: Error: When you define paddingStart you should probably also define paddingEnd for right-to-left symmetry [RtlSymmetry]
215
        android:paddingStart="@dimen/switch_text_padding"
216
        ~~~~~~~~~~~~~~~~~~~~
```
when committing do not bypass githooks, they are there to let you know a PR will fail at this step.

Reply:
@macgills Is it fine now?
![sc](https://user-images.githubusercontent.com/41234408/73464901-9663a900-43a5-11ea-8152-80a544c49c44.png)

Reply:
there shouldn't be that much space between the text of the switch and the actual switch, use `android:layout_gravity="end"` and that should produce the result I am looking for

Reply:
But excellent, it looks like the dark theme worked there, you can probably remove the textColor attribute now as the ThemeOverlay should be contributing it

Reply:
Ok. 
I thought we just wanted the switch part to go to the end. Apologies for interpreting it in the wrong way. I'll fix it right now.

Reply:
@macgills is it fine now?
![sc](https://user-images.githubusercontent.com/41234408/73465764-de370000-43a6-11ea-8a35-13fa8c1b4482.png)

Reply:
The search/delete icons aren't present in that image? But it is looking pretty good. Touch more padding on the bottom and to the right.
Also whenever we show txt we should use `android:textAppearance=?textAppearance*` where `*` should be one of the following
```
<!--Material type attributes-->
    <item name="textAppearanceHeadline1">@style/TextAppearance.KiwixTheme.Headline1</item>
    <item name="textAppearanceHeadline2">@style/TextAppearance.KiwixTheme.Headline2</item>
    <item name="textAppearanceHeadline3">@style/TextAppearance.KiwixTheme.Headline3</item>
    <item name="textAppearanceHeadline4">@style/TextAppearance.KiwixTheme.Headline4</item>
    <item name="textAppearanceHeadline5">@style/TextAppearance.KiwixTheme.Headline5</item>
    <item name="textAppearanceHeadline6">@style/TextAppearance.KiwixTheme.Headline6</item>
    <item name="textAppearanceSubtitle1">@style/TextAppearance.KiwixTheme.Subtitle1</item>
    <item name="textAppearanceSubtitle2">@style/TextAppearance.KiwixTheme.Subtitle2</item>
    <item name="textAppearanceBody1">@style/TextAppearance.KiwixTheme.Body1</item>
    <item name="textAppearanceBody2">@style/TextAppearance.KiwixTheme.Body2</item>
    <item name="textAppearanceButton">@style/TextAppearance.KiwixTheme.Button</item>
    <item name="textAppearanceCaption">@style/TextAppearance.KiwixTheme.Caption</item>
    <item name="textAppearanceOverline">@style/TextAppearance.KiwixTheme.Overline</item>
```
more info [here](https://material.io/design/typography/the-type-system.html#type-scale)

Reply:
@macgills Since there is no history in the screenshot I took, the search and delete icons are not visible. When history is there, the icons will be present.
I'll make changes to the text appearance.

Reply:
@macgills made the changes. 
Also putting in a screenshot of how it looks when history is present.
![sc](https://user-images.githubusercontent.com/41234408/73467443-56062a00-43a9-11ea-9ed8-c1a19fcc58a6.png)

Reply:
We want to get rid of the overflow menu when we have enough space, in the menu.xml use `IfRoom` for the delete icon instead of never, also supply the trash icon

Reply:
@macgills done :rocket:

Reply:
Thanks for all the hard work